src/components/LocationSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Location } from '../types/sparkeats';
+
+type LocationSelectProps = {
+  locations: Location[];
+};
+
+function LocationSelect({ locations }: LocationSelectProps) {
+  const allCities = locations.map((location) => location.city);
+  const uniqueCities = [...new Set(allCities)];
```

Comment:
This is great, but can we move the to [locations.ts](https://github.com/sparkbox/sparkeats/blob/main/src/locations.ts)? The main reason is to keep the logic and data derived from it coming from a single source and Sparkeats doesn't have enough for this to make a significant difference in performance or maintenance. This way only need to pass the component the prop that it needs.

Reply:
Updated in 7e978d4abef13ec8f47773f39e520aea155e109b.